refugeerouter/models.py

```diff
@@ -19,15 +19,29 @@ def __str__(self):
 
 
 class Refugee(models.Model):
+    GENDER_DIVERSE='D'
+    GENDER_FEMALE='F'
+    GENDER_MALE='M'
+    GENDER_UNKNOWN='U'
+    GENDER_CHOICES = [
+        (GENDER_DIVERSE, 'Diverse'),
+        (GENDER_FEMALE, 'Female'),
+        (GENDER_MALE, 'Male'),
+        (GENDER_UNKNOWN, 'Unknown'),
+    ]
     id = models.UUIDField(default=uuid.uuid4, primary_key=True)
     first_name = models.CharField(max_length=1024)
     last_name = models.CharField(max_length=1024)
-    age = models.IntegerField(default=1)
-    gender = models.IntegerField(default=1)
+    age = models.IntegerField(default=40)
+    gender = models.CharField(
+        max_length=1,
+        choices=GENDER_CHOICES,
+        default=GENDER_UNKNOWN
+    )
     contact_data = models.CharField(max_length=1024)
     origin = models.CharField(max_length=1024)
     origin_checked = models.BooleanField(default=False)
-    group = models.ForeignKey(Group, on_delete=models.CASCADE)
+    group = models.ForeignKey(Group, on_delete=models.CASCADE, blank=True, null=True)
```

Comment:
ah, missed that, refugees without a group doesn't make any sense. so we should always enforce one

Reply:
My thought was: what if a refugee is moved out of a group. This one should not have a group.